DRUID.Rmd

```diff
@@ -7,6 +7,7 @@ author:
   url: null
 date: "`r Sys.Date()`"
```

Comment:
Is it possible to have both a "Published" and a "Last Updated" Date? Would be great to pin the "published" date to the actual date it goes live, and then Last Updated to `r Sys.Date()`, but this is just a minor point.